examples/upload-media-from-local/README.md

```diff
@@ -0,0 +1,35 @@
+# Upload media from local
+
+This is a script to handle uploading media locally to Studio.
```

Comment:
I would rather say: This is a script that can upload media files to Studio from local.

Reply:
done